community-events/content/challenge-project/00-analyzing-portal.md

```diff
@@ -0,0 +1,73 @@
+# 🔎 Exercise 1: Analyzing the `portal` project
+
+The `portal` of the Contoso Real Estate project is a simulation of a real estate agency where you can view the properties available for sale, rent, or buy. The architecture of the project was designed in a composable architecture.
+
+<img src="./images/composable-architecture-example.png" title="Composable Architecture Diagram" height="450" />
+
+Composable architecture is a software architecture that divides the application into smaller parts, where each part is responsible for a specific functionality. This architecture is also known as micro frontends.
+
+**Advantages of using composable architecture:**
+
+- **Reusability**: components can be reused in different parts of the application.
+- **Scalability**: the application can be scaled horizontally, that is, it can be divided into smaller parts, and each part can be developed by a different team.
+- **Maintainability**: the application can be maintained more easily as each component is responsible for a specific functionality.
+
+## Project architecture
+
+The Contoso Real Estate project is composed of three main parts:
+
+- `portal`: is the main project, where it contains the home page and the components that will be used on all pages.
+- `blog`: is the project that contains the real estate blog. In this project, `Strapi` is used as a CMS to manage blog posts.
+- `api`: is the project that contains the APIs that will be used in the `portal` project.
+
+![Contoso Real Estate Diagram](./images/e2e-full-horizontal.drawio.png)
+
+### The `portal` components
+
+This project was developed using Angular, and you will learn from execution to deployment using the Azure Static Web Apps CLI in Codespaces.
+
+Accessing the `packages/portal/src/app` folder, you will see the components that make up the `portal`:
+
+- `app`: main component of the application, responsible for rendering the other components.
+  - `about`: about page
+  - `authentication`: login page (authentication)
+  - `checkoutpage`: checkout page
+  - `core`: component responsible for writing data to the browser (local storage)
```

Comment:
`core` folder has been deprecated since a few Angular version, as it's not needed anymore. Any services located in there should be moved to `shared`, it probably came from a PR.

Reply:
Okay, I'll remove `core` from the list